Makefile

```diff
@@ -453,35 +456,38 @@ fixtures/rpm-unsigned-modified: fixtures
 	createrepo --update $@
 
 fixtures/rpm-packages-updateinfo: fixtures
-	rpm/gen-patched-fixtures.sh $@ rpm/updateinfo-packages.patch
+	rpm/gen-patched-fixtures.sh -d $@ -f rpm/updateinfo-packages.patch
 
 fixtures/rpm-references-updateinfo: fixtures
-	rpm/gen-patched-fixtures.sh $@ rpm/references-updateinfo.patch
+	rpm/gen-patched-fixtures.sh -d $@ -f rpm/references-updateinfo.patch
 
 fixtures/rpm-richnweak-deps: fixtures/srpm-richnweak-deps fixtures
 	rpm-richnweak-deps/gen-rpms.sh $@ $</*.src.rpm
 
 fixtures/rpm-updated-updateinfo: fixtures
-	rpm/gen-patched-fixtures.sh $@ rpm/updated-updateinfo.patch
+	rpm/gen-patched-fixtures.sh -d $@ -f rpm/updated-updateinfo.patch
 
 fixtures/rpm-updated-updateversion: fixtures
-	rpm/gen-patched-fixtures.sh $@ rpm/updated-updateversion.patch
+	rpm/gen-patched-fixtures.sh -d $@ -f rpm/updated-updateversion.patch
 
 fixtures/rpm-with-md5: fixtures
 	rpm/gen-fixtures.sh --checksum-type "md5" $@ rpm/assets
 
 fixtures/rpm-with-modules: fixtures
-	rpm/gen-patched-fixtures.sh $@ rpm/modules-updateinfo.patch
+	rpm/gen-patched-fixtures.sh -d $@ -f rpm/modules-updateinfo.patch
 	modifyrepo --mdtype=modules rpm/assets/modules.yaml "$@/repodata/"
 
 fixtures/rpm-with-modules-modified: fixtures
-	rpm/gen-patched-fixtures.sh $@ rpm/modules-updateinfo.patch
+	rpm/gen-patched-fixtures.sh -d $@ -f rpm/modules-updateinfo.patch
 	rm $@/kangaroo-0.3-1.noarch.rpm
 	rm $@/shark-0.1-1.noarch.rpm
 	rm $@/stork-0.12-2.noarch.rpm
 	createrepo --update $@
 	modifyrepo --mdtype=modules rpm/assets/modules.yaml "$@/repodata/"
 
+fixtures/rpm-modules-static-context: fixtures
+	rpm/gen-patched-fixtures.sh -d $@ -f rpm/modules-static-context.patch -t module
```

Comment:
My understanding is that it will result in the modules.yaml with mixed format, aka some modules have static_context, some do not. Correct?  It's good, quite a realistic situation when folks start mixing up content from different repos.

Reply:
Yes - that's deliiberate on my part. I don't see anything in the spec that forbids it, so I figure it'll happen eventually!